juniper_warp/scripts/rustc_version_is_supported.sh

```diff
@@ -0,0 +1,9 @@
+# Exits successfully if the Rust version that is running is compatible with juniper_warp.
```

Comment:
We don't need this file anymore, right?

Reply:
Oh definitely, my bad. I'll also try to see if we can revert to using cargo make with the new feature.

Reply:
We can do that in a follow-up, just delete this file for now.

Reply:
done